<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataUnitTypeEdit</class>
 <widget class="QWidget" name="DataUnitTypeEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>134</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>value type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="boolBtn">
        <property name="text">
         <string>boolean</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="singleBtn">
        <property name="text">
         <string>single value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="arrayBtn">
        <property name="text">
         <string>array</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dateBtn">
        <property name="text">
         <string>date</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>dimension</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="dimEdit">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
